en/controllers/pagination.rst

```diff
@@ -139,39 +124,16 @@ this count query can be very expensive. In situations where you only want to
 show 'Next' and 'Previous' links you can use the 'simple' paginator which does
 not do a count query::
 
-    public function initialize(): void
+    class ArticlesController extends AppController
     {
-        parent::initialize();
-
-        // Load the paginator component with the simple paginator strategy.
-        $this->loadComponent('Paginator', [
-            'paginator' => new \Cake\Datasource\SimplePaginator(),
-        ]);
+        protected array $paginate = [
+            'paginator' => \Cake\Datasource\SimplePaginator::class,
+        ];
     }
 
 When using the ``SimplePaginator`` you will not be able to generate page
 numbers, counter data, links to the last page, or total record count controls.
 
-Using the PaginatorComponent Directly
```

Comment:
Should we have a section on how to use Paginator directly?

Reply:
Sorry, I missed your comment and forgot about this PR. Will look into it soon.